TYPO3.Media/Resources/Private/Templates/Asset/ReplaceAssetResource.html

```diff
@@ -0,0 +1,59 @@
+{namespace m=TYPO3\Media\ViewHelpers}
```

Comment:
just like 80% of the other labels in the file, probably best to create an issue to check all those templates